### PR TITLE
LibPDF: Fix text/line matrices not being retroactively applied to font size + using LibGfx::ICC

### PR DESCRIPTION
**Pull request by mattco98**
Page 1 of the PDF 2.0 spec:

<details><summary>Before</summary>

</details> 

<details><summary>After</summary>

</details> 

cc @nico for the ICC stuff. Turns out the ICC changes don't actually do anything to contribute to the images above. I thought it did, and then implemented it all only to find out I misinterpreted the color information I was seeing from the PDF. But if it looks ok, might as well merge it.

### CODE REVIEW

File: Userland/Libraries/LibGfx/ICC/WellKnownProfiles.h [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 #include <AK/Error.h>
 #include <AK/NonnullRefPtr.h>
+#include <LibGfx/ICC/TagTypes.h>
```
Comment (nico) [inline Userland/Libraries/LibGfx/ICC/WellKnownProfiles.h:11]: Can we forward declare TagData next to Profile 4 lines further down instead?

File: Userland/Libraries/LibPDF/ColorSpace.cpp [hunk 2]
```diff
@@ -303,39 +306,66 @@ PDFErrorOr<NonnullRefPtr<ColorSpace>> ICCBasedColorSpace::create(Document* docum
...
-Color ICCBasedColorSpace::color(Vector<Value> const&) const
+ICCBasedColorSpace::ICCBasedColorSpace(NonnullRefPtr<Gfx::ICC::Profile> profile)
+    : m_profile(profile)
 {
-    VERIFY_NOT_REACHED();
+}
+
+Color ICCBasedColorSpace::color(Vector<Value> const& arguments) const
+{
+    if (!s_srgb_profile)
```
Comment (nico) [inline Userland/Libraries/LibPDF/ColorSpace.cpp:334]: This is alright for now, but eventually the target profile should be passed in to Render, and the GUI should pass in the color space of the window's backing bitmap, and WindowServer should give each bitmap the color space belonging to the monitor the window is on and every color and bitmap should be associated with some color space and painter should do the conversion transparently etc etc etc. Obviously not for this PR 😅

Comment (mattco98) [reply Userland/Libraries/LibPDF/ColorSpace.cpp:334]: Yeahhh I think I'll leave that one for you :smiley:

**Comment by mattco98** [comment]
Just realized I forgot the `PPI/DEFAULT_DPI` constant when setting the font size. The "After" picture looks even better now!

**Pull request merged**